src/dwn-server.ts

```diff
@@ -1,37 +1,33 @@
 import type { Config } from './config.js';
 
-import { Dwn, DataStoreLevel, EventLogLevel, MessageStoreLevel } from '@tbd54566975/dwn-sdk-js';
+import { Dwn } from '@tbd54566975/dwn-sdk-js';
 
 import { WsApi } from './ws-api.js';
 import { HttpApi } from './http-api.js';
 import { config as defaultConfig } from './config.js';
 import { HttpServerShutdownHandler } from './lib/http-server-shutdown-handler.js';
+import { getDWNConfig } from './storage.js';
 
 export type DwnServerOptions = {
   dwn?: Dwn;
   config?: Config;
 };
 
 export class DwnServer {
-  dwn: Dwn;
+  dwn?: Dwn;
   config: Config;
   httpServerShutdownHandler: HttpServerShutdownHandler;
 
   constructor(options: DwnServerOptions = {}) {
     this.config = options.config ?? defaultConfig;
-    this.dwn = options.dwn;
+    if(options.dwn) {
```

Comment:
is there a reason for this change or you just adding suger?

Reply:
there is not, i'll make this go back to how it was. I was comparing to previous iterations of this PR not main, in this diff it's now obvious that this is silly.